apps/dashboard/src/utils/validation-schema.ts

```diff
@@ -79,6 +83,19 @@ export const createPayoutSchema =
     user: yup.mixed<BaseUserResponse>().required(),
   });
 
+export const createProductSchema =
+    yup.object({
+        name: yup.string().required(),
+        priceInclVat: yup.number().required().default(0),
+        vat: yup.mixed<BaseVatGroupResponse>().required(),
+        category: yup.mixed<ProductCategoryResponse>().required(),
+        alcoholPercentage: yup.number().required().default(0),
+        featured: yup.boolean().required().default(false),
```

Comment:
featured, preferred and priceList are not required  by the backend.

Reply:
Hahahaha fair point!